src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -266,25 +293,25 @@ def __init__(self, parent=None):
         self._bothAuto.toggled[bool].connect(self.__bothToggled)
         self._bothAuto.setFocusPolicy(qt.Qt.TabFocus)
 
-        self._minAuto = qt.QCheckBox(self)
-        self._minAuto.setText("")
+        self._minAuto = qt.QPushButton(self)
+        self._minAuto.setText("Min")
         self._minAuto.setToolTip("Enable/disable the autoscale for min")
+        self._minAuto.setCheckable(True)
         self._minAuto.toggled[bool].connect(self.__minToggled)
         self._minAuto.setFocusPolicy(qt.Qt.TabFocus)
+        self._minAuto.setVisible(False)
```

Comment:
Unless I missed something, `self._minAuto` can be completely removed, it is never shown.

Reply:
Yes, but i am afraid of breaking external stuffs using private stuffs.

Reply:
That's a private attribute in a private class instantiated as a private attribute.... Using it externally is looking for trouble.

I made a PR on your branch for it.